server/base/src/main/java/org/apache/accumulo/server/constraints/MetadataConstraints.java

```diff
@@ -107,6 +108,7 @@ private static boolean isValidColumn(ColumnUpdate cu) {
 
   @Override
   public List<Short> check(Environment env, Mutation mutation) {
+    context = ((SystemEnvironment)env).getServerContext();
```

Comment:
I think it would be a little cleaner to pass the context to `getArbitrator()` and not have it as an instance var.  The test that overrides  `getArbitrator()` could just ignore the context.